protoc-gen-gogo/generator/generator.go

```diff
@@ -2071,124 +2071,129 @@ func (g *Generator) generateMessage(message *Descriptor) {
 		}
 	}
 
-	for i, field := range message.Field {
-		// Allocate the getter and the field at the same time so name
```

Comment:
Fixed in next push. I intended to fix it, but I rather do that if I have time in the future in a different PR.